tests/integration/test_users.py

```diff
@@ -39,6 +45,13 @@ def test_user_can_follow_another_user(client, users_repo):
     assert 'Adam' in followed_users_names
 
 
+def test_user_follow_another_user_400_error(client):
+    response = client.post('/users/1/follow', data=json.dumps({
+        'user_pk': '2',
+    }), content_type='application/json')
+    assert response.status_code == 400
+
+
```

Comment:
Testing every endpoint for `400` errors will become repetitive, but until we have a general solution (like a base `MarshmallowResource` handling errors) I think it's ok.